src/config.ts

```diff
@@ -89,6 +89,32 @@ export default {
   DEFAULT_AUDIO_TRACK_SWITCHING_MODE: "seamless" as "seamless" |
                                                     "direct",
 
+  /**
+   * In some cases after switching the current track or bitrate, the RxPlayer
+   * could be led to go into the `"RELOADING"` state, which corresponds to
+   * visually a black screen (with nothing audible) before restarting playback.
+   *
+   * We could want to seek back some milliseconds when doing that.
+   * For example, when switching the current audio track, it might make sense
+   * to restart some time before, so the beginning of the sentence can be heard
+   * again in the new language.
+   *
+   * This config property allows to set the relative position the RxPlayer will
+   * seek to after reloading, in seconds.
+   *
+   * For example: a value of `-0.7` means that will seek back 700 milliseconds
+   * when reloading due to a track or bitrate switch with necessitated a
+   * reloading.
+   */
+  DELTA_POSITION_AFTER_RELOAD: {
+    /** Relative position when switching the bitrate */
+    bitrateSwitch: -0.1,
+    /** Relative position when switching the track */
+    trackSwitch: { audio: -0.7,
```

Comment:
Maybe we could have the same delta for each type.

Reply:
After doing tests, I noticed that seeking back after switching the video track (like the camera) was much less pleasant.
And the concept of giving back concept is less present with video.

Reply:
Ok. Maybe it could be nice to write it in the comment above.